Tests/StreamChatTests/Utils/Logger_Tests.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+import Foundation
+@testable import StreamChat
+import XCTest
+
+final class Logger_Tests: XCTestCase {
+    func test_log_isThreadSafe() {
+        LogConfig.destinationTypes = [ConsoleLogDestination.self]
+
+        DispatchQueue.concurrentPerform(iterations: 100) { _ in
```

Comment:
Without your changes, is those tests failing?

Reply:
Yes 👍